test/swagger.spec.js

```diff
@@ -101,331 +112,257 @@ describe('Swagger docs', function () {
-                            tags: undefined
+                            type: 'number'
                         }
                     }
                 },
                 relationships: {
-                    id: 'relationships',
                     type: 'object',
                     properties: {
                         pets: {
-                            type: 'pets',
-                            defaultValue: null,
-                            description: undefined,
-                            notes: undefined,
-                            tags: undefined
+                            '$ref': '#/definitions/pets',
+                            type: 'object'
                         },
                         soulmate: {
-                            type: 'soulmate',
-                            defaultValue: null,
-                            description: undefined,
-                            notes: undefined,
-                            tags: undefined
+                            '$ref': '#/definitions/soulmate',
+                            type: 'object'
                         }
                     }
                 },
                 data: {
-                    id: 'data',
                     type: 'object',
+                    required: ['id', 'type'],
                     properties: {
                         id: {
                             type: 'string',
-                            required: true,
-                            defaultValue: undefined,
-                            description: 'RFC4122 v4 UUID',
-                            notes: undefined,
-                            tags: undefined
+                            description: 'RFC4122 v4 UUID'
                         },
                         type: {
                             type: 'string',
-                            required: true,
-                            defaultValue: undefined,
-                            enum: ['pets'],
-                            description: undefined,
-                            notes: undefined,
-                            tags: undefined
+                            enum: ['pets']
                         }
                     }
                 }
             }
             _.forEach(models, function (item, key) {
-                expect(result.models[key]).to.eql(item)
+                expect(definitions[key]).to.eql(item)
             })
         })
         it('should describe GET /people parameters', function () {
-            const api = apis['GET /people'].operations[0]
+            const api = apis['/people'].get
             const parameters = {}
             _.forEach(api.parameters, function (item) {
                 parameters[item.name] = item
             })
             expect(parameters).to.have.property('include')
-            expect(parameters.include).to.have.property('paramType', 'query')
+            expect(parameters.include).to.have.property('in', 'query')
```

Comment:
Hey @ShurakaiSoft, could you fill me in a bit here. Did the swagger spec output change between hapi-swagger versions? how come the prop changed from 'paramType' to 'in'